sugarcrm/__init__.py

```diff
@@ -0,0 +1,14 @@
+import os
+import logging
+
+rest_url = os.getenv("SUGARCRM_REST_URL")
+username = os.getenv("SUGARCRM_USER")
+password = os.getenv("SUGARCRM_PASSWORD")
+membership_roles = os.getenv("SUGARCRM_MEMBERSHIP_ROLES")
+if membership_roles:
+    membership_roles = membership_roles.split(",")
+logger = logging.getLogger("django.server")
+
+
+class CrmError(Exception):
```

Comment:
I'd rather move this to a `sugarcrm/errors.py` so as not to mix too many concerns in this file.